docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -110,37 +110,54 @@ \section{Proof of Concept Demonstration Plan}
 
 \section{Expected Technology}
 
-\wss{What programming language or languages do you expect to use?  What external
-libraries?  What frameworks?  What technologies.  Are there major components of
-the implementation that you expect you will implement, despite the existence of
-libraries that provide the required functionality.  For projects with machine
-learning, will you use pre-trained models, or be training your own model?  }
-
-\wss{The implementation decisions can, and likely will, change over the course
-of the project.  The initial documentation should be written in an abstract way;
-it should be agnostic of the implementation choices, unless the implementation
-choices are project constraints.  However, recording our initial thoughts on
-implementation helps understand the challenge level and feasibility of a
-project.  It may also help with early identification of areas where project
-members will need to augment their training.}
-
-Topics to discuss include the following:
+% \wss{What programming language or languages do you expect to use?  What external
+% libraries?  What frameworks?  What technologies.  Are there major components of
+% the implementation that you expect you will implement, despite the existence of
+% libraries that provide the required functionality.  For projects with machine
+% learning, will you use pre-trained models, or be training your own model?  }
+
+These are the following expected technologies that the team expects to use for the project:
 
 \begin{itemize}
```

Comment:
we can also add a test coverage tools in our CI pipeline like what they have [here](https://github.com/Eric-Jalal/cpp-pipeline/blob/develop/.github/workflows/build.yml#L30-L33). `gcov` seems to be the most used but you can def. look into it more.